### DISCUSSION TITLE
What is the maximum number of concurrently connected clients in EMQ X?

### DISCUSSION THREAD

**Discussion by eleleme** [emqx/emqx#5266] [General]
BrokerX单机最大并发连接的客户端多大？如果要支持10万设备连接需要多少个节点？

**Comment by Rory-Z** [comment]
> EMQ X is designed for massive clients access and realizes fast and low-latency message routing between massive physical network devices:
> 
> 1. Stable to host large-scale MQTT client connections, and a single-server node supports millions of connections.
 
Please read our [documentation](https://docs.emqx.io/en/broker/v4.3/)
